### PR TITLE
Wrap required arguments up together

### PR DESCRIPTION
**Pull request by erbridge**
For now we only have `input`, but this allows us to change the API in future and matches the API more directly.

Builds on #18 to avoid conflicts.

### CODE REVIEW

File: lib/Model.js [hunk 1]
```diff
@@ -122,12 +122,16 @@ export default class Model extends ReplicateObject {
     return prediction;
   }
 
-  async createPrediction(input, { webhook, webhookEventsFilter } = {}) {
+  async createPrediction({ input }, { webhook, webhookEventsFilter } = {}) {
```
Comment (zeke) [inline lib/Model.js:125]: Why does it take two objects? Couldn't this just be one object that looks something like `{input = {}, webhook, webhookEventsFilter}` ?

**Comment by zeke** [comment]
(I changed the base branch on the PR to keep the redundant commits out and make this easier to review.)

File: lib/Model.js (hunk 1 above)
Comment (erbridge) [reply lib/Model.js:125]: I personally like the interface where the arguments to the prediction are separated from the optional configuration arguments. And I see webhooks as the latter, not the former.

Comment (zeke) [reply lib/Model.js:125]: What arguments are there to the prediction other than `input`?

```js
createPrediction({input, whatElseWouldGoHere?}, {webhook})
```

Comment (erbridge) [reply lib/Model.js:125]: Nothing yet, but plausibly something in the future, and given `input` is an object with arbitrary keys, I'm not sure how we'd switch from that being the only required argument to there being a `whatElseWouldGoHere` without a new method or people's stuff breaking in subtle ways after a package upgrade.

**Review by zeke — APPROVED** [review]
Let's give this a go and see how it feels to use it.

**Pull request merged**